generate_tim.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+# This script is used to generate a TIM (Traveler Information Message) from a JSON file using the
+# Operational Data Environment (ODE). It sends the JSON file to the ODE, retrieves the TIM from the ODE logs,
+# extracts the HEX value, and saves it to a file with the same name as the input file but with a .uper extension.
+
+
+if [ $# -ne 1 ]; then
+    echo "Usage: ./generate_tim.sh <filename>"
+    exit 1
+fi
+
+filename=$1
+
+# retrieve IP address of the host machine
+export DOCKER_HOST_IP=$(ifconfig | grep -A 1 'inet ' | grep -v 'inet6\|127.0.0.1' | awk '{print $2}' | grep -E '^172\.1[6-9]\.|^172\.2[0-9]\.|^172\.3[0-1]\.|^192\.168\.' | head -n 1)
```

Comment:
Should we try to pull from the .env first in case someone is running this script against a different ip?

Reply:
Yup sounds like a good idea. Modified this script to use the .env if it exists and to create one if it doesn't, just like the other script.